api/handler/acl.go

```diff
@@ -167,6 +167,8 @@ func (h *handler) GetBucketACLHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	h.log.Debug("target details", zap.String("cid", bktInfo.CID.String()))
```

Comment:
Isn't it too excessive to log every request processing (event successful)?

Reply:
It should be OK for debug, NeoGo can log every P2P packet in this mode. But I'd rather use `zap.Stringer` here (and everywhere down below) to avoid `.String()` transformation for every request.

Reply:
@masterSplinter01 logging `cid` and (if applicable) `oid` on every call is the idea of the issue. However, the `debug` level saves us from doings this all the time.
@roman-khimov do you recommend the Stringer because of the specification? `Stringer constructs a field with the given key and the output of the value's String method. The Stringer's String method is called lazily.`

Reply:
To call `h.log.Debug()` we need its parameters, one of them is the result of `zap.String()` that takes two arguments, one of which is the result of `bktInfo.CID.String()` -> `String()` is always called in this case, with some associated allocation/base58 overhead.

Now with `Stringer()` we only have to convert `bktInfo.CID` to interface to get parameters, it's much easier to do that. And then IIUC zap is smart enough to check the message level first and then extract data from fields only if it's needed (and call `String()` in this case).

Granted, all of this probably isn't critical at the moment, but still we better use more optimal APIs when they're available (this inspired nspcc-dev/neo-go#2842 BTW).

Reply:
I thought that in most cases printing of `oid` and `cid` is needed when something goes wrong. But ok :)

Reply:
I thought that in most cases printing of oid and cid is needed when something goes wrong. But ok :)